addons/addon-base-raas/packages/base-raas-services/lib/schema/create-egress-store.json

```diff
@@ -100,7 +100,6 @@
     "rev",
     "inWorkflow",
     "status",
```

Comment:
With Jeet's change of removing `CIDR` from AppStream enabled accounts, we don't always pass in `cidr`. Therefore we need to remove it from here, or it will cause a validation error.

Reply:
Can cidr me completely removed? I wonder why egress store needs it.